pkg/strc/stgraph/builder.go

```diff
@@ -105,7 +105,7 @@ func (b *TraceBuilder) Process(r io.Reader, w io.Writer, format string) error {
 		rdy := b.Window.Append(&span)
 		if rdy {
 			spans := b.Window.Pop(span.TraceID)
-			if spans == nil || len(spans) < 1 {
+			if len(spans) < 1 {
```

Comment:
Unrelated, I was rushing with the `stgraph` CLI tool to hit a demo timeline and I pushed without running tests the other day :-)